doctor/models.py

```
@@ -0,0 +1,595 @@
+        verbose_name='پزشک'
+    )
+    
+    WEEKDAY_CHOICES = [
+        (0, 'شنبه'),
+        (1, 'یکشنبه'),
+        (2, 'دوشنبه'),
+        (3, 'سه‌شنبه'),
+        (4, 'چهارشنبه'),
+        (5, 'پنج‌شنبه'),
+        (6, 'جمعه'),
+    ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)